internal/magic/zip.go

```diff
@@ -59,6 +59,23 @@ func Jar(raw []byte, limit uint32) bool {
 		}}, 1)
 }
 
+// Kmz matches a zipped KML file, which should be "doc.kml" by convention but it
+// should be enough if extension matches.
+func Kmz(raw []byte, _ uint32) bool {
+	iter := zipIterator{raw}
+	for {
+		f := iter.next()
+		if len(f) == 0 {
+			break
+		}
+		if bytes.HasSuffix(f, []byte(".kml")) {
```

Comment:
I think it's better to match exactly `doc.kml`. Just the extension might give false positives.

Reply:
Fixed.